core/src/main/java/io/parsingdata/metal/expression/value/ConstantFactory.java

```diff
@@ -29,8 +29,12 @@
 
     private ConstantFactory() {}
 
+    public static Slice makeConstantSlice(final byte[] data) {
```

Comment:
Good idea, I've renamed it to `createFromBytes()`. Additionally, I figured that `ConstantFactory` actually exclusively produces `Value` instances, so I moved it to `Util` instead.